versions.html

```diff
@@ -20,20 +20,20 @@ <h1> PyTorch Documentation </h1>
         <li class="toctree-l1">
           <a class="reference internal" href="main/">main (unstable)</a>
         </li>
+        <li class="toctree-l1">
+          <a class="reference internal" href="0.13.0/">v0.9.0 (release candidate)</a>
+        </li>
         <li class="toctree-l1">
           <a class="reference internal" href="0.12.0/">v0.12.0 (stable release)</a>
         </li>
-       <li class="toctree-l1">
+        <li class="toctree-l1">
           <a class="reference internal" href="0.11.0/">v0.11.0</a>
         </li>
-       <li class="toctree-l1">
-          <a class="reference internal" href="0.10.0/">v0.10.0</a>
-        </li>
         <li class="toctree-l1">
-          <a class="reference internal" href="0.9.0/">v0.9.0</a>
+          <a class="reference internal" href="0.10.0/">v0.10.0</a>
         </li>
       </ul>
     </div>
     </div></div>
   </body>
-</html>
+</html>
```

Comment:
looks like the newline is removed from the end?

Reply:
Added